charts/gateway/values.yaml

```diff
@@ -564,13 +564,14 @@ config:
       ssl:
         enabled: false
         truststore:
+          existingSecret:
```

Comment:
Can we just call it secret?

Reply:
Add a section in readme for GemFire existingSecret would be helpful